bigquery/biglake/biglake_metastore_create_table/main.tf

```diff
@@ -0,0 +1,123 @@
+/**
+* Copyright 2023 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+* This Terraform code sample creates a BigLake Metastore catalog, database, and
+* table. For more information, see
+* https://cloud.google.com/bigquery/docs/manage-open-source-metadata and
+* https://cloud.google.com/bigquery/docs/reference/biglake/rest
+*/
+
+# [START biglake_metastore_create_table]
+
+# This creates a BigLake Metastore in the US region named "my-catalog".
+# BigLake Metastore catalogs can contain multiple databases.
+resource "google_biglake_catalog" "catalog" {
+  name     = "my_catalog"
+  location = "US"
+}
+
+# Create a Cloud Storage Bucket with a unique name in the `US`.
+# This creates a bucket in the US region named "my-bucket" with a pseudorandom
+# suffix.
+resource "random_id" "bucket_name_suffix" {
+  byte_length = 8
+}
+resource "google_storage_bucket" "bucket" {
+  name                        = "my-bucket-${random_id.bucket_name_suffix.hex}"
+  location                    = "US"
+  force_destroy               = true
+  uniform_bucket_level_access = true
+}
+
+# This creates a Google Cloud Storage object to store metadata.
+resource "google_storage_bucket_object" "metadata_directory" {
+  name    = "metadata/"
+  content = " "
+  bucket  = google_storage_bucket.bucket.name
+}
+
+# This creates a Google Cloud Storage object to store data.
+resource "google_storage_bucket_object" "data_directory" {
+  name    = "data/"
+  content = " "
+  bucket  = google_storage_bucket.bucket.name
+}
+
+# This creates a BigLake Metastore database with the name "my-database" and the
+# type "HIVE".
+# The database is created in the catalog specified by the
+# "google_biglake_catalog.catalog.id" variable.
```

Comment:
How is "This creates ... type "HIVE" in the catalog specified by ... variable." (similarly below)

Also "my-database" or "my_database"?